data/language/en-GB.txt

```diff
@@ -3646,6 +3646,11 @@ STR_6387    :Can't lower element here…
 STR_6388    :Can't raise element here…
 STR_6389    :Invalid clearance
 
+# Picking RCT2 directory 
```

Comment:
We don't typically put these kind of comments in the language file, to avoid making it too big. (Also note that many strings are scattered instead of grouped together, which is another reason adding comments won't really work well.)